release/.releaserc.js

```diff
@@ -0,0 +1,129 @@
+module.exports = {
+    "branches": [
+        "main",
+        {
+            "name": "test",
```

Comment:
You have only `develop` and `main` branches in your workflow, do you want to change the workflow?

Reply:
I changed the branches, can you check again?

Reply:
Added develop branch for prereleases